src/Config/RabbitMQExtensionConfigProvider.cs

```diff
@@ -2,27 +2,36 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.Json;
 using System.Text;
 using Microsoft.Azure.WebJobs.Description;
+using Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Trigger;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
 
 namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
 {
     [Extension("RabbitMQ")]
     internal class RabbitMQExtensionConfigProvider : IExtensionConfigProvider
     {
         private readonly IOptions<RabbitMQOptions> _options;
+        private readonly IConfiguration _configuration;
+        private readonly INameResolver _nameResolver;
         private readonly IRabbitMQServiceFactory _rabbitMQServiceFactory;
         private readonly ILoggerFactory _loggerFactory;
+        private ILogger _logger;
 
-        public RabbitMQExtensionConfigProvider(IOptions<RabbitMQOptions> options, IRabbitMQServiceFactory rabbitMQServiceFactory, ILoggerFactory loggerFactory)
+        public RabbitMQExtensionConfigProvider(IOptions<RabbitMQOptions> options, IConfiguration configuration, IRabbitMQServiceFactory rabbitMQServiceFactory, ILoggerFactory loggerFactory)
         {

```

Comment:
Should it  be INameResolver nameResolver instead of IConfiguration configuration?